x-pack/plugins/apm/server/tutorial/index.ts

```diff
@@ -109,13 +97,5 @@ It allows you to monitor the performance of thousands of applications in real ti
         cloudSetup: cloud,
       }),
       previewImagePath: '/plugins/apm/assets/apm.png',
-      savedObjects,
-      savedObjectsInstallMsg: i18n.translate(
-        'xpack.apm.tutorial.specProvider.savedObjectsInstallMsg',
-        {
-          defaultMessage:
-            'An APM data view is required for some features in the APM UI.',
-        }
-      ),
```

Comment:
I decided to remove the option of building the data view directly from the tutorial. This was only supported via the legacy saved object calls, and no longer adds any value, since the data view is automatically generated when opening APM UI.